src/Internal/Support/StackRenderer.php

```diff
@@ -11,6 +11,24 @@
 
 class StackRenderer
 {
+    /**
+     * Sets files and prefixes to be ignored from the stack trace.
+     *
+     * @var array
+     */
+    private static array $ignorePaths = [
+        'temporal/sdk/src/Internal/',
```

Comment:
Maybe the namespace makes sense? Instead of path i mean.

Reply:
Does not handle Closures.